test/tabs_test.js

```diff
@@ -93,6 +93,7 @@ describe('Tabs', function() {
 
   describe('selecting new tab (2)', function() {
     before(function(done) {
+      const browser = new Browser();
```

Comment:
It's not closing the tab that triggers the event! As you said, it's triggered by changing focus using **browser.tabs.current = 0;** and that tab has a focus event from the previous test. Even if you close the tab, the event gets attached to the newly opened  one.